zerver/tests/test_docs.py

```diff
@@ -413,6 +413,36 @@ def test_github_production_error(self) -> None:
         self.assert_not_in_success_response(["zproject/dev_settings.py"], result)
         self.assert_not_in_success_response(["zproject/dev-secrets.conf"], result)
 
+    @override_settings(SOCIAL_AUTH_GITLAB_KEY=None)
+    def test_gitlab(self) -> None:
```

Comment:
I think this test was meant to be named something like ``test_gitlab_development_error``?

Reply:
The similar tests have the same naming flaw; we can just fix in a new commit.